src/lib/accessibility/Behaviors/Icon/iconBehavior.ts

```diff
@@ -2,8 +2,9 @@ import { Accessibility } from '../../types'
 
 /**
  * @description
```

Comment:
it seems that we can omit @description tag altogether, as it is redundant. It will look like:

```js
/**
 * Adds attribute 'aria-hidden=true', if there is no 'alt' property provided.	
 * Image is usually only visual representation and therefore is hidden from screen readers.	 * Image is usually only visual representation and therefore is hidden from screen readers.
 * 
 * @specification
 * Adds attribute 'aria-hidden=true', if there is no 'alt' property provided.
 */
```
What do you think?